pyproject.toml

```diff
@@ -11,7 +11,7 @@ cached-property = "~=1.4"
 cdiserrors = "~=1.0"
 xmltodict = "~=0.9"
 
-authlib = "~=0.11"
+authlib = "<0.14.3"
```

Comment:
We should not do this if `authutils` itself is not broken by authlib 0.14.3, or we're building a multi-repo monolith.